packages/opam.rb

```diff
@@ -7,68 +7,65 @@ class Opam < Package
   description 'OCaml package manager'
   homepage 'https://opam.ocaml.org/'
   @_ver = '2.1.1'
-  version @_ver
+  version "#{@_ver}-2"
   compatibility 'all'
   source_url 'https://github.com/ocaml/opam.git'
   git_hashtag @_ver
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1_armv7l/opam-2.1.1-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1_armv7l/opam-2.1.1-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1_i686/opam-2.1.1-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1_x86_64/opam-2.1.1-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1-2_armv7l/opam-2.1.1-2-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1-2_armv7l/opam-2.1.1-2-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1-2_i686/opam-2.1.1-2-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/opam/2.1.1-2_x86_64/opam-2.1.1-2-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: '53d3b4e436c1a70f5772523f0af5454aa54ad7dfb777d50fe7495d31160b7462',
-     armv7l: '53d3b4e436c1a70f5772523f0af5454aa54ad7dfb777d50fe7495d31160b7462',
-       i686: '4d8e3979729315249b0a9dabf795108c91d16618dd5f38285dc758eabf3896d1',
-     x86_64: 'f0cced8d7504be9b5bbd2f98d346e28b9cd18cd8b02985722643854a5cd5d51f'
+    aarch64: 'ef542a60a56abda5549b9e0a07e76898315dd07149bda008f0498b5244ec79b8',
+     armv7l: 'ef542a60a56abda5549b9e0a07e76898315dd07149bda008f0498b5244ec79b8',
+       i686: '380e81c2ce976bf42ccfc18050598c50bf282b157a269f4dd179d0410ab2dbb9',
+     x86_64: 'cb95bae566579fcf5d1952afe41c89d40932a10458e90c00a0f812ae9ef0f43f'
   })
 
   depends_on 'bubblewrap'
   depends_on 'ocaml'
   depends_on 'rsync'
 
+  @OPAMROOT = "#{CREW_PREFIX}/share/opam"
+
   def self.build
     system "#{CREW_ENV_OPTIONS} ./configure #{CREW_OPTIONS}"
     system "make lib-ext all -j1 \
       OCAMLC='ocamlc -unsafe-string' \
       OCAMLOPT='ocamlopt -unsafe-string'"
-    @bashd_opam = <<~EOF
-      export OPAMROOT=#{CREW_PREFIX}/opam
-      eval \$(opam env --root=#{CREW_PREFIX}/opam --switch=default)
-      test -r #{CREW_PREFIX}/opam/opam-init/init.sh && . #{CREW_PREFIX}/opam/opam-init/init.sh > /dev/null 2> /dev/null || true
-    EOF
+    @bashd_opam = <<~OPAMEOF
+      export OPAMROOT=#{@OPAMROOT}
+      eval \$(opam env --root=#{@OPAMROOT} --switch=default)
+      test -r #{@OPAMROOT}/opam-init/init.sh && . #{@OPAMROOT}/opam-init/init.sh > /dev/null 2> /dev/null || true
```

Comment:
`&> /dev/null`?

Reply:
This is intentionally echoing the format in that file:
```
cat /usr/local/share/opam/opam-init/init.sh
if [ -t 0 ]; then
  test -r /usr/local/share/opam/opam-init/complete.sh && . /usr/local/share/opam/opam-init/complete.sh > /dev/null 2> /dev/null || true

  test -r /usr/local/share/opam/opam-init/env_hook.sh && . /usr/local/share/opam/opam-init/env_hook.sh > /dev/null 2> /dev/null || true
fi

test -r /usr/local/share/opam/opam-init/variables.sh && . /usr/local/share/opam/opam-init/variables.sh > /dev/null 2> /dev/null || true
```

Reply:
I think what @supechicken means is you can replace `> /dev/null 2> /dev/null` with just `&> /dev/null`.

Reply:
Understood. I'll respin.

Reply:
new binaries uploaded.